### ISSUE TITLE
Carousel component - Support for embedding other sql files

### ISSUE THREAD

**Issue by accforgithubtest** [lovasoa/SQLpage#605]
> What is your problem ? A description of the problem, not the solution you are proposing.

I have built a dashboard with multiple cards in the view. There are many cards that the page is scrolling vertically now.
I would like for this data to be displayed as a dashboard in a "kiosk" fashion (i.e - no manual interaction expected), where by all the data is displayed to the user, in a auto-scrolling / slideshow format.

I checked out the `carousel` component, but it seems to only support images currently. 
Only `cards` component seems to support embedding other `.sql` files. however cards doesn't have any features to auto-scroll / slideshow the data.

>  Describe the solution you'd like

It would be really nice if the carousel component can also embed other .sql files, like the cards component, and show the output of the embedded sql files as a "slideshow".
This would allow for the carousel component to really add the potential for dashboard displays that can be used in "kiosk" mode.

Hope my explanation of the request is clear / makes sense. 
Thanks for your time and consideration !

**Comment by lovasoa** [comment]
Hello, that would indeed be useful. And this will probably be easy to implement after we merge https://github.com/lovasoa/SQLpage/pull/615

What do you think, @mtt-artis ?

**Comment by mtt-artis** [comment]
The embed option is already there on the server side so it might be doable just by transforming the carousel template with an `embed` option. Or am I missing something?

Should we explain the `data-embed="SRC"` tricks in the doc for custom template ?

**Comment by lovasoa** [comment]
Yes, that's what I was saying, the feature asked for here should be easy to implement after we merge your pull request.

I think we should document data-embed in [/examples/official-site/custom_components.sql](https://github.com/lovasoa/SQLpage/blob/main/examples/official-site/custom_components.sql)
